modules/dashboard/test/TestPlan.md

```diff
@@ -46,32 +46,26 @@
 14. Verify that if a user has 'issue_tracker_reporter / issue_tracker_developer' permission,
     the issue tracker panel should not be found without this permission.
     [Automate Test on Travis_CI]
-15. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
-    account approvals is displayed in the My Task panel. This should be the number of entries in the User Account
-    page with the following Selection Filter: Site set to the user's site and Pending Approval set to 'Yes'. The
-    Site displayed will be 'All user sites'. Check that you are taken to that page (with the Selection Filter
-    correctly set) when you click on the task.
-    [Automate Test on Travis_CI]
-16. Verify that a user with 'Violated Scans: View all-sites Violated Scans' permission has a task with the number
+15. Verify that a user with 'Violated Scans: View all-sites Violated Scans' permission has a task with the number
```

Comment:
@nicolasbrossard What's the larger discussion about moving all of these widget points under their respective modules? (e.g. violated scans).  Has there been a decision that we should all be doing this?

Reply:
@christinerogers Not sure there was a larger discussion regarding this. Dave requested this change: you might want to check with him.

Reply:
The dashboard test plan was written before the module used any widgets, and no longer houses the code being tested. There are many things you can do now which affect what shows up in the dashboard without the dashboard knowing anything about it.. write a new module for a project which has widgets, disable/enable an existing module in the module manager, override a (non-dashboard) module which is one of the ones that provides widgets..

The tickets came up when I made tickets for the `candidate_profile` widgets that are missing documentation since the `dashboard` and `candidate_profile` use the same `getWidgets()` function.

Reply:
Thanks @driusan. 
Sounds like the Dashboard Readme needs updating with exactly this info, no?